kolibri/core/exams/migrations/0001_initial.py

```diff
@@ -25,7 +25,7 @@ class Migration(migrations.Migration):
                 ('_morango_source_id', models.CharField(editable=False, max_length=96)),
                 ('_morango_partition', models.CharField(editable=False, max_length=128)),
                 ('title', models.CharField(max_length=200)),
-                ('channel_id', models.CharField(max_length=32)),
+                ('channel_id', models.CharField(max_length=32, blank=True)),
```

Comment:
Does this work with an existing database?

Reply:
Actually, is this even necessary if you are removing the field?

Reply:
I did this as a convenience because I was toggling back and forth between branches. It's not necessary